lib/tasks/work_tasks.rake

```diff
@@ -25,4 +25,14 @@ namespace :work do
     end
   end
 
+  # AO3-6065
+  desc "Add 'Not Rated' to works missing a rating"
+  task(:not_rated => :environment) do
+    Work.find_each do |work|
+      if work.rating_string.blank?
```

Comment:
We could probably avoid tucking everything inside an `if...end` by doing `next unless work.rating_string.blank?` or `next if work.rating_string.present?` instead.